common_design.theme

```diff
@@ -65,7 +64,7 @@ function common_design_form_alter(&$form, FormStateInterface $form_state, $form_
     if ($form_id == 'views_exposed_form') {
```

Comment:
You don't need this line anymore, only the in_array one

Reply:
+1